src/v0/util/googleUtils/index.js

```diff
@@ -92,12 +98,14 @@ const finaliseAnalyticsConsents = (consentConfigMap, eventLevelConsent = {}) =>
     // Set consent only if valid
     if (
       eventLevelConsent &&
-      eventLevelConsent.hasOwnProperty(consentKey) &&
-      GA4_ALLOWED_CONSENT_STATUS.includes(eventLevelConsent[consentKey])
+      eventLevelConsent.hasOwnProperty(configKey) &&
```

Comment:
user is expected to provide consent keys as per destination format itself `ad_user_data` --> we are not generalising them

Reply:
addressed